source/module_base/opt_DCsrch.cpp

```diff
@@ -171,18 +171,18 @@ int dcsrch(double& stp,
     double ginit = 0.0, gtest = 0.0, gm = 0.0, gx = 0.0, gxm = 0.0, gy = 0.0, gym = 0.0;
     double stx = 0.0, sty = 0.0, stmin = 0.0, stmax = 0.0, width = 0.0, width1 = 0.0;
 
-    extern /* Subroutine */ void dcstep(double&,
-                                        double&,
-                                        double&,
-                                        double&,
-                                        double&,
-                                        double&,
-                                        double&,
-                                        double&,
-                                        double&,
-                                        bool&,
-                                        double&,
-                                        double&);
+    extern /* Subroutine */ void dcstep(double& /*stx*/,
```

Comment:
looks strange here

Reply:
Add variable names according to declaration to improve code readability.

Reply:
We don't recommend writing "/* */"